FlowCrypt/Controllers/Compose/ComposeDecorator.swift

```diff
@@ -59,6 +60,34 @@ struct ComposeDecorator: ComposeDecoratorType {
             insets: UIEdgeInsets(top: 8, left: 8, bottom: 8, right: 8)
         )
     }
+
+    func styledReplyQuote(with input: ComposeViewController.Input) -> NSAttributedString {
+        guard case let .reply(info) = input.type else { return NSAttributedString(string: "") }
+
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateStyle = .short
+        dateFormatter.timeStyle = .none
+
+        let date = dateFormatter.string(from: info.sentDate)
+
+        dateFormatter.dateStyle = .none
+        dateFormatter.timeStyle = .short
+        let time = dateFormatter.string(from: info.sentDate)
+
+        let from = info.recipient?.displayName
+            ?? info.recipient?.mailbox
+            ?? "no subject"
+
+        let text: String = "\n\n"
+            + "compose_reply_from_title".localized
```

Comment:
Don't add this title, it's not needed

Reply:
`Reply from iOS` is not needed? It was in task description. Removed

Reply:
That was supposed to be example of a message reply that the user is sending.

If I receive a message `This was the original message\nNow it is quoted`, and I type type `reply from iOS`, then the sent message should be exactly as defined in the issue:
```
reply from iOS

On 2019-11-21 at 12:11, mindoutofframe@gmail.com wrote:
> This was the original message
> Now it is quoted
```